From 10e223e3a09ae96aef3f522facee9d8c21b135a8 Mon Sep 17 00:00:00 2001
From: Jurgen Kramer <gtmkramer@xs4all.nl>
Date: Tue, 10 Oct 2017 08:57:34 +0200
Subject: [PATCH] Native DSD support for Matrix Audio X-SPDIF 2

This patch adds native DSD support for the Matrix Audio X-SPDIF 2.

Signed-off-by: Jurgen Kramer <gtmkramer@xs4all.nl>
---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index 9aeb05f..a8ac8ce 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1344,6 +1344,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 
 	/* XMOS based USB DACs */
 	switch (chip->usb_id) {
+	case USB_ID(0x20b1, 0x2004): /* Matrix Audio X-SPDIF 2 */
 	case USB_ID(0x20b1, 0x3008): /* iFi Audio micro/nano iDSD */
 	case USB_ID(0x20b1, 0x2008): /* Matrix Audio X-Sabre */
 	case USB_ID(0x20b1, 0x300a): /* Matrix Audio Mini-i Pro */
-- 
2.9.5

